en-US/firefox-ios.xliff

```diff
@@ -4138,6 +4138,28 @@ Private Tab</target>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/Footer.strings" datatype="plaintext" source-language="en" target-language="en-US">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="14.0.1" build-num="14A400"/>
+    </header>
+    <body>
+      <trans-unit id="FirefoxHomepage.Pocket.Footer.LearnMore.v115" xml:space="preserve">
+        <source>Learn more</source>
+        <target>Learn more</target>
+        <note>This is the learn more text of the Pocket footer on Firefox Homepage.</note>
+      </trans-unit>
+      <trans-unit id="FirefoxHomepage.Pocket.Footer.Subtitle.v115" xml:space="preserve">
+        <source>Part of the %@ family.</source>
+        <target>Part of the %@ family.</target>
+        <note>This is the subtitle of the Pocket footer on Firefox Homepage. The placeholder is the app name.</note>
+      </trans-unit>
+      <trans-unit id="FirefoxHomepage.Pocket.Footer.Title.v115" xml:space="preserve">
+        <source>Powered by Pocket.</source>
+        <target>Powered by Pocket.</target>
```

Comment:
@adudenamedruby Sorry, it seems like my comment hadn't been posted here yesterday like I thought. 
I was saying that "Pocket" should be a placeholder since it is a brand